tensorboard/components/tf_ng_tensorboard/reducers.ts

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {InjectionToken} from '@angular/core';
```

Comment:
File level comment: I know we still have the intrastructure to work out. But does it make sense to include a test for this?

Reply:
Thanks for the suggestion! Diff I thought would grow and thought I'd add it only when reducer get more complex with more business logic but it was so simple! Done